web/admin/src/buttons.js

```diff
@@ -19,19 +19,19 @@ import './App.css'
 
 export default class CustomButtons extends Component {
   render() {
-    const { markBlock, report, unBlock, currentKey, userId } = this.props
+    const { markBlock, report, unBlock, currentKey, currentUser} = this.props
     if (markBlock) {
       return (
         <span className='cellBoxRight'>
-          <img className='button1' onClick={() => markBlock(report, currentKey, userId)} src={require('./icons/deleteocircle.svg')} alt="block" />
-          <img className='button1' onClick={() => unBlock(report, currentKey, userId)} src={require('./icons/checkocircle.svg')} alt="check" />
+          <img className='button1' onClick={() => unBlock(report, currentKey, currentUser)} src={require('./icons/checkocircle.svg')} alt="check" />
```

Comment:
Since we are using ES6 syntax, remove these `require` calls and add `import` statements at the top of the file.